src/jit/optcse.cpp

```diff
@@ -98,7 +102,7 @@ void Compiler::optUnmarkCSE(GenTree* tree)
     // make sure it's been initialized
```

Comment:
`CSEdsc* desc;` is now only used in the `if` so maybe sink the declaration down there too.